### PR TITLE
Improve wording of README

### PR DESCRIPTION
**Pull request by llawall**
Make it clear that Apache Kafka is the product name and the streams implemented in Python is a Kafka API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # Winton Kafka Streams
 
-Implementation of [Apache Kafka Streams](https://kafka.apache.org/documentation/streams/) in Python. 
+Implementation of [Apache Kafka's streams API](https://kafka.apache.org/documentation/streams/) in Python.
```
Comment (miguno) [inline README.md]: "streams API" -> "Streams API"

File: README.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
-Implementation of [Apache Kafka Streams](https://kafka.apache.org/documentation/streams/) in Python. 
+Implementation of [Apache Kafka's streams API](https://kafka.apache.org/documentation/streams/) in Python. 
 
 ## What and why?
 Apache Kafka is an open-source stream processing platform developed
 by the Apache Software Foundation written in Scala and Java. Kafka
-Streams is a Java library for building distributed stream processing
-applications using Apache Kafka. Kafka Streams does not require any
+has streams API added for building stream processing applications
```
Comment (miguno) [inline README.md]: Same here (streams API -> Streams API).  Will not mention it again in this PR. :-)

File: README.md (hunk 1 above)
Comment (ah-) [inline README.md]: Cool, thanks! Will update in a bit.

File: README.md (hunk 2 above)
Comment (llawall) [inline README.md]: https://github.com/wintoncode/winton-kafka-streams/pull/5

Above PR should fix the case.

**Pull request merged**